locations/spiders/visionexpress_gb.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+import scrapy
+from locations.linked_data_parser import LinkedDataParser
+from locations.google_url import url_to_coords
+
+
+def extract_google_position(item, response):
+    for link in response.xpath("//img/@src").extract():
+        if link.startswith("https://maps.googleapis.com/maps/api/staticmap"):
+            item["lat"], item["lon"] = url_to_coords(link)
+            return
+
+
+def set_located_in(item, located_in, located_in_wikidata):
```

Comment:
This is now in 2 spiders, should it be moved to `items.py`?

Reply:
It's on my list but not for this commit.

Reply:
I still don't understand this pattern. It doesn't make sense to extract two assignments into a function. Is there extra functionality you're thinking would go in here in the future?

I could see an argument to be made for better readability, but the misdirection makes it harder to follow than putting the assignments in line in my opinion.